### PR TITLE
updated sync icon and migration progress

### PR DESCRIPTION
**Pull request by criticalAY**
## Pull Request template
Added migration progress icon in the menu

## Purpose / Description
Puts the migration icon while we migrate and hides the sync icon

## Fixes
follow up for #13130

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by david-allison** [comment]
```
> Task :AnkiDroid:ktlintMainSourceSetCheck FAILED
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt:700:11 Missing spacing after "if"
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt:720:1 Needless blank line(s)
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt:820:77 Missing spacing after ","
```

lint failure - aware you're working through issues now

**Comment by criticalAY** [comment]
@david-allison Have fixed the code

**Review by david-allison — CHANGES_REQUESTED** [review]
The arrow seems slightly smaller, confirm I'm correct (didn't screw up the layered screenshots), then fix, then LGTM

Screenshot 2023-02-23 at 20 31 19

**Comment by criticalAY** [comment]
I have increased the icon size by little

File: AnkiDroid/src/main/res/layout/migration_progress_menu_layout.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ~  This program is distributed in the hope that it will be useful, but WITHOUT ANY
+  ~  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+  ~  PARTICULAR PURPOSE. See the GNU General Public License for more details.
+  ~
+  ~  You should have received a copy of the GNU General Public License along with
+  ~  this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
```
Comment (david-allison) [inline AnkiDroid/src/main/res/layout/migration_progress_menu_layout.xml:18]: Timebox an investigation to see why this is needed (both start and end are 12dp too short)

```suggestion
    android:layout_height="wrap_content"
    android:paddingHorizontal="12dp"
```

File: AnkiDroid/src/main/res/layout/migration_progress_menu_layout.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    android:clickable="true">
+
+    <ProgressBar
+        android:id="@+id/ic_progressDonut"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="28dp"
+        android:layout_height="28dp"
```
Comment (david-allison) [inline AnkiDroid/src/main/res/layout/migration_progress_menu_layout.xml:25]:
```suggestion
        android:layout_width="30dp"
        android:layout_height="30dp"
```

**Review by david-allison — CHANGES_REQUESTED** [review]
Nitpicks to make this pixel perfect. One request to find out why this is necessary (timeboxed) then happy to squash + go.

Thanks!

**Review by david-allison — CHANGES_REQUESTED** [review]
marking as needing feedback - albeit minor.

**Comment by criticalAY** [comment]
I get it what was missing here, Thanks David!
[ The icon itself has a width of 12dp. Whole button’s width is 36dp: (12dp + (2 * 12dp) = 36dp).](https://medium.com/@lucasurbas/making-android-toolbar-responsive-2627d4e07129#)

**Comment by criticalAY** [comment]
I haved added the title in this menu item do we need that here or are we okay without it?

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
> I haven't added the title in this menu item do we need that here or are we okay without it?

Useful, if you have time

**Comment by criticalAY** [comment]
I added the title and tested but it look like it doesn't work that way with action layouts so we are good to go now

**Comment by david-allison** [comment]
> I added the title and tested but it look like it doesn't work that way with action layouts so we are good to go now

Drop a TODO for later

**Review by mikehardy — APPROVED** [review]
Nice!

**Pull request merged**